### PR TITLE
Add alias support for icon helper.

### PR DESCRIPTION
**Pull request by demiankatz**
As discussed on #1962, this adds the ability to create icon aliases.

### CODE REVIEW

**Review by EreMaijala — APPROVED** [review]
Looks good to me!

**Comment by demiankatz** [comment]
Thanks, @EreMaijala! I'll wait for @crhallberg to take a look before merging, just in case he has a different opinion regarding naming, etc.

@crhallberg, if you're happy with this as-is, feel free to hit the merge button. :-)

**Review by crhallberg — APPROVED** [review]

**Pull request merged**
